### ISSUE TITLE
Cover now gives error code 400

### ISSUE THREAD

**Issue by milkmanrox** [hatl/hasscontrol#58]
I've had a garage cover entity working for ages, but now it gives "Failed unknown error code 400".

Cover entity looks like this:

```
cover:
  - platform: template
    covers:
      garage_door:
        device_class: garage
        friendly_name: "Garage Door"
        value_template: >-
          {% if is_state('binary_sensor.shelly_plus_1_garage_door_switch_0_input','off') %}
            Open
          {% else %}
            Closed
          {% endif %}
        open_cover:
          service: switch.turn_on
          data:
            entity_id: switch.shelly_plus_1_garage_door_switch_0
        close_cover:
          service: switch.turn_on
          data:
            entity_id: switch.shelly_plus_1_garage_door_switch_0
        stop_cover:
          service: switch.turn_on
          data:
            entity_id: switch.shelly_plus_1_garage_door_switch_0
        icon_template: >-
          {% if is_state('binary_sensor.shelly_plus_1_garage_door_switch_0_input','off') %}
            mdi:garage-open
          {% else %}
            mdi:garage
          {% endif %}
```

I've got a group with both the cover entity and the actual switch entity that the cover refers to, and the latter works but not the former.

**Comment by alexander-th** [comment]
I can confirm the same behavior with the following cover configuration:

`cover:
  - platform: template
    covers:
      grosse_garage:
        device_class: garage
        friendly_name: "große Garage"
        position_template: >-
          {% if is_state("binary_sensor.grosse_garage_geschlossen", "on") %} 
            0
          {% elif is_state("binary_sensor.grosse_garage_offen", "on") %} 
            100 
          {% else %}
            50 
          {% endif %}     
        open_cover:
          - condition: state
            entity_id: binary_sensor.grosse_garage_offen
            state: "off"
          - service: switch.turn_on
            target:
              entity_id: switch.garage_gross_tor
        close_cover:
          - condition: state
            entity_id: binary_sensor.grosse_garage_geschlossen
            state: "off"
          - service: switch.turn_on
            target:
              entity_id: switch.garage_gross_tor
        stop_cover:
          service: switch.turn_on
          target:
            entity_id: switch.garage_gross_tor
        icon_template: >-
          {% if is_state("binary_sensor.grosse_garage_geschlossen","on")  %}
            mdi:garage
          {% else %}
            mdi:garage-open
          {% endif %}`

**Comment by hatl** [comment]
fixed in v1.3.1
